pages/vi/profiles/jessewashburn.md

```diff
@@ -0,0 +1,20 @@
+# Jesse Washburn  
+
+👋 Name| 🕖 Time Zone | 🖥️ OS|
+------|-----------|-------
+Jesse Washburn | Eastern U.S | Windows 11 version 23H2
+
+### About me:  
+
+Hi! My name is Jesse. I am a 3rd year Computer Science student at Towson University, which is near Baltimore, MD. 🇺🇸 I am originally from Pori, Finland.  🇫🇮 
+
+My interests are web/mobile development, data analytics, and SMB/enterprise software. In my free time, I like to play classical guitar. I'm looking forward to getting to know the OLE team and learning from you all as a **Software Engineering Intern.**   
+
+Let's connect via [Github][2] or [LinkedIn][1]!  
+
+Cheers! 
+
+Jesse
+
+[1]: https://www.linkedin.com/feed/        "LinkedIn"
```

Comment:
Did you intend to use this link or it was an oversight?

Reply:
My apologies. The link has been corrected.